src/renderer/features/wallets-select/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { createStore } from 'effector';
+
+import { createFeature } from '@/shared/effector';
+import { navigationHeader } from '@/features/app-shell';
+import { SelectWalletPairing, WalletSelect } from '@/features/wallets';
+
+export const walletsSelectFeature = createFeature({
+  name: 'Wallets select',
+  enable: createStore(true),
+});
+
+walletsSelectFeature.inject(navigationHeader, () => {
```

Comment:
Just a notice. Inject has no difference between inject pipeline and slot. During development we should check what we inject, or validation will says us that we have incorrect second argument. I am not sure that we should do something with it, but it can confuse a little. Use `Slot`suffix (`navigationHeaderSlot`) maybe can fix it

Reply:
Agree, just stumbled upon this when came to look `.tsx` file

Reply:
The difference between identifiers are backed into types, which inferred into handler (2nd arg). Naming convention maybe a good decision and we should also fix it into our lint rules.